src/main/resources/db/schema/SL-964-set-all-transactions-to-comitted.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
```

Comment:
I'm not sure if it should reside inside schema package, because it's not changing the schema, but data.